shared/common-adapters/box.js.flow

```diff
@@ -5,7 +5,7 @@ export type Props = any
 
 export type Box2Props = {|
   alignItems?: 'center' | 'flex-start' | 'flex-end',
-  alignSelf?: null | 'flex-start' | 'flex-end',
+  alignSelf?: null | 'center' | 'flex-start' | 'flex-end',
```

Comment:
i believe we didn't do this cause you're already centered if you're not full width/height